tests/console/ndctl.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: install ndctl. Destroy and configure NVDIMM namespaces
+# Maintainer: Alvaro Carvajal <acarvajal@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils qw(zypper_call);
+
+sub run {
+    my $self = shift;
+    return unless get_var('NVDIMM');
+    $self->select_serial_terminal;
+    zypper_call('in ndctl');
+    assert_script_run 'ndctl destroy-namespace --force all';
+    my $total = get_var('NVDIMM_NAMESPACES_TOTAL', 2);
+    foreach my $i (0 .. ($total - 1)) {
+        my $device = script_output 'ndctl create-namespace --mode=fsdax --map=dev';
+	$device =~ /\"blockdev\":\"(pmem\d+)\"/;
+        assert_script_run "ls /dev/$device";
+        assert_script_run "wipefs -a /dev/$device";
+    }
+    assert_script_run 'ndctl list';
```

Comment:
I assume we would need something like 

`ndctl create-namespace --force --reconfig=namespace0.0 --mode=memory --map=dev`

and 

`ndctl list -H -N -D -F` 

to ckeck for FW of the DIMMS just to make sure FW side of things are okey

Reply:
Output of `ndctl list -H -N -D -F` is really long to be seen in a single screen without scrolling. Added the command in the test with the output redirected to a file and then this file will be uploaded. Information will be in the Logs & Assets tab of the test.